pokemongo_bot/cell_workers/incubate_eggs.py

```diff
@@ -176,6 +180,9 @@ def _check_inventory(self, lookup_ids=[]):
                             pokemon.get('individual_stamina', 0)
                         ]})
                     matched_pokemon.append(pokemon)
+                    #remove as egg and add as pokemon
+                    inventory.pokemons().remove(pokemon['id'])
+                    inventory.pokemons().add(inventory.Pokemon(pokemon))
```

Comment:
"It seems like it is going to work like that. It is hard to be entirely sure since this file is a maze." I know right?